src/main/java/org/sonarlint/intellij/ui/SonarLintWalkthroughToolWindow.java

```diff
@@ -0,0 +1,482 @@
+
+    var closeButton = new JButton("Close");
+    var closeButtonPanel = new JPanel(new FlowLayout(FlowLayout.RIGHT));
+    closeButtonPanel.add(closeButton);
+
+    gbc.anchor = GridBagConstraints.SOUTHWEST;
+
+    provideCommonButtonConstraints(gbc);
+
+    page4CenterPanel.add(backButtonPanel, gbc);
+
+    gbc.anchor = GridBagConstraints.SOUTHEAST;
+
+    page4CenterPanel.add(closeButtonPanel, gbc);
+
+    panel.add(imageLabel, BorderLayout.NORTH);
+    panel.add(page4CenterPanel, BorderLayout.CENTER);
+
+    closeButton.addActionListener(e -> {
+      var toolWindow = ToolWindowManager.getInstance(project).getToolWindow("Welcome to SonarQube for IDE");
+      if (toolWindow != null) {
+        toolWindow.hide(null);
+      }
+    });
+  }
+
+  private static @NotNull JEditorPane createReachOutToUsPageText(Font font, Project project) {
+    var descriptionPane = new JEditorPane(EDITOR_PANE_TYPE, "<html><body style=\"font-family: " + font.getFamily() +
+      "; font-size: " + font.getSize() + "pt; padding: 5px;\">" +
+      "You suspect any issue with " + SONARQUBE_FOR_IDE + "? Check the <a href=\"#logView\">log view</a>.<br>" +
+      "Share the verbose logs with us via <a href=\"#communityForum\">Community forum</a> in case of problem. We will be happy to help " +
+      "you debug.<br><br>" +
+      "Learn more about " + SONARQUBE_FOR_IDE + " through <a href=\"#docs\">our docs</a>.</body></html>");
+
+    descriptionPane.setEditable(false);
+    descriptionPane.setOpaque(false);
+    descriptionPane.addHyperlinkListener(new HyperlinkAdapter() {
+      @Override
+      public void hyperlinkActivated(HyperlinkEvent e) {
+        if ("#logView".equals(e.getDescription())) {
+          var sonarqubeToolWindow = ToolWindowManager.getInstance(project).getToolWindow(SONARQUBE_FOR_IDE);
+          if (sonarqubeToolWindow != null) {
+            if (!sonarqubeToolWindow.isVisible()) {
+              sonarqubeToolWindow.activate(null);
+            }
+            var currentFileContent = sonarqubeToolWindow.getContentManager().findContent("Log");
+            if (currentFileContent != null) {
+              sonarqubeToolWindow.getContentManager().setSelectedContent(currentFileContent);
+            }
+          }
+        } else if ("#communityForum".equals(e.getDescription())) {
+          BrowserUtil.browse("https://community.sonarsource.com/c/sl/fault/6");
+        } else if ("#docs".equals(e.getDescription())) {
+          BASE_DOCS_PAGE.browseWithTelemetry();
+        }
+      }
+    });
+
+    return descriptionPane;
+  }
+
+  private static @NotNull JEditorPane createLearnAsYouCodePageText(Font font, Project project) {
+    var descriptionPane = new JEditorPane(EDITOR_PANE_TYPE, "<html><body style=\"font-family: " + font.getFamily() +
+      "; font-size: " + font.getSize() + "pt; padding: 5px;\">" +
+      "Check the <a href=\"#currentFile\">Current File</a> view: When " + SONARQUBE_FOR_IDE + " found something, click on the issue to " +
```

Comment:
```suggestion
      "Check the <a href=\"#currentFile\">Current File</a> view: When " + SONARQUBE_FOR_IDE + " finds something, click on the issue to " +
```
Or "once SonarQube for IDE found something"